test/libsolidity/SolidityCompiler.cpp

```diff
@@ -45,12 +45,12 @@ BOOST_AUTO_TEST_CASE(does_not_include_creation_time_only_internal_functions)
 	m_compiler.setOptimiserSettings(dev::test::Options::get().optimize);
 	BOOST_REQUIRE(success(sourceCode));
 	BOOST_REQUIRE_MESSAGE(m_compiler.compile(), "Compiling contract failed");
-	bytes const& creationBytecode = m_compiler.object("C").bytecode;
-	bytes const& runtimeBytecode = m_compiler.runtimeObject("C").bytecode;
-	BOOST_CHECK(creationBytecode.size() >= 130);
-	BOOST_CHECK(creationBytecode.size() <= 160);
-	BOOST_CHECK(runtimeBytecode.size() >= 50);
-	BOOST_CHECK(runtimeBytecode.size() <= 70);
+	bytes const& creationBytecode = dev::test::bytecodeSansMetadata(m_compiler.object("C").bytecode);
+	bytes const& runtimeBytecode = dev::test::bytecodeSansMetadata(m_compiler.runtimeObject("C").bytecode);
+	BOOST_CHECK(creationBytecode.size() >= 90);
+	BOOST_CHECK(creationBytecode.size() <= 120);
+	BOOST_CHECK(runtimeBytecode.size() >= 10);
+	BOOST_CHECK(runtimeBytecode.size() <= 30);
```

Comment:
Subtracted 40 from each of these numbers, given the metadata CBOR is: 2+1+1+5+1+32 bytes long (42 bytes).